api/service/networkinfo/service.go

```diff
@@ -114,27 +113,32 @@ func (s *Service) Run() {
 
 // DoService does network info.
 func (s *Service) DoService() {
+	_, ipv4Net, err := net.ParseCIDR("100.64.0.0/10")
```

Comment:
Also for the future reference, [`netaddr.MustNewIPNetwork()`](https://godoc.org/github.com/xgfone/netaddr#MustNewIPNetwork) is useful when we parse a string literal into an IP prefix: Since the input is a literal, it will always succeed or always fail, and if it always fails, might as well panic early (especially if the variable is declared at package level) because that's most likely a bug (like a typo).